types/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "module": "es2015",
+    "lib": ["es2015"],
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
```

Could this be folded into https://www.typescriptlang.org/tsconfig#strict ?
```suggestion
    "strict": true,
```